lib/mpl_toolkits/axes_grid1/tests/test_axes_grid1.py

```diff
@@ -707,3 +707,36 @@ def test_removal():
     fig.canvas.draw()
     col.remove()
     fig.canvas.draw()
+
+
+@image_comparison(['anchored_locator_base_call.png'], style="mpl20")
```

Comment:
```suggestion
@image_comparison(['anchored_locator_base_call.png'], style="mpl20", remove_text=True)
```
Or is the text a crucial part of the appearance? 

(It is generally a good idea to remove the text if not needed, plus that it is cropped in this example.)

(A good approach to avoid conflicts is to insert the test at some random location in the file. This is as many insert it at the end and therefore it is more likely that there will be conflicts there. Hopefully this will be merged soon though.)

Reply:
@oscargus. Thank you for the feedback. I agree with your comments but i had trouble generating a test image that would pass the image comparison test when remove_text is true. I do agree, however, that the test should be for just the image and not the text. I have resubmitted the test case where I didn't put the "remove_text" but did remove all the text in the test case and generate a test image without any text. 

If this isn't quite acceptable, can you provide more details on your proposed approach? Thanks!